### DISCUSSION TITLE
Prompt Icon

### DISCUSSION THREAD

**Discussion by jolevich** [warbacon/zunder-zsh#10] [Q&A] [answered]
Hi! I've been loving your zsh config, but I'm wondering how I could change the thunder prompt icon?

**Comment by warbacon** [comment, marked as answer]
I'm glad you like it! :)

Zunder-zsh uses my own custom prompt called [zunder-prompt](https://github.com/Warbacon/zunder-prompt). I created it to be as simple and fast as possible; it doesn't even have customization options yet. The thing is, what you're asking for isn't that hard to implement and shouldn't affect performance. I also think it makes sense to implement it. I will implement it in zunder-prompt in the coming days and provide instructions in the project's readme on how to change the icon and its color (you should run the ``zi update`` command to update the plugin when it happens).

In the meantime, you can manually edit the plugin or even fork it. It's not too complex.

Feel free to reach out if you have any other questions!

**Reply by warbacon** [reply]
Okay, so it's already implemented, it really didn't take long. You have the instructions in the zunder-zsh readme and also the zunder-prompt readme.

Remember to update the plugin using ``zi update``.

Best regards!

**Reply by jolevich** [reply]
Working like a charm, thank you!
